docs/manual.rst

```diff
@@ -2329,7 +2329,7 @@ filters.typogrify_custom
     ``[typo.amp, typo.widont, typo.smartypants, typo.caps, typo.initial_quotes]`` and the
     default value for ``ignore_tags`` is ``["title", ".math"]``. If ``ignore_tags`` is
     specified, the default tags will be appended to the supplied list. See the
-    `documentation <https://github.com/mintchaos/typogrify/blob/master/typogrify/filters.py#L8-L14>`__
+    `documentation <https://github.com/justinmayer/typogrify/blob/master/typogrify/filters.py#L8-L14>`__
```

Comment:
I think the line numbers need to be adjusted. I guess for that one first has to do some git archeology to figure out what these lines linked to in the past...

Reply:
As far as I can tell, they never linked to something very specific, perhaps we should just remove the line numbers? I just copied the one we had in there and made sure they point to the same text. Let me know what you think.